packages/edit-site/src/components/sidebar-dataviews/default-views.js

```diff
@@ -29,25 +29,18 @@ import {
 export const defaultLayouts = {
 	[ LAYOUT_TABLE ]: {
 		layout: {
-			primaryField: 'title',
 			styles: {
-				title: {
+				__primary: {
```

Comment:
I see primary has been renamed to `titleField`. I like that better. Can we follow here the convention and use something along the lines of `__title`?

Reply:
I think this is probably not working for now. I'm going to remove this until we have a valid use-case for it.